lib/system/services.toit

```diff
@@ -323,6 +335,7 @@ class ServiceResourceProxyManager_ implements SystemMessageHandler_:
 
 class ServiceManager_ implements SystemMessageHandler_:
   static instance := ServiceManager_
+  static uninitialized/bool := true
```

Comment:
?
```suggestion
  static initialized/bool := false
```

Reply:
I think I can avoid the whole thing in a follow-up. It just requires some surgery on the rpc test.